### PR TITLE
Tests for reactions

### PR DESCRIPTION
**Pull request by lafriks**
Tests for #2856

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: models/issue_reaction_test.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```
Comment (ethantkoenig) [inline models/issue_reaction_test.go]: import order

**Comment by lafriks** [comment]
@ethantkoenig fixed

**Review by ethantkoenig — APPROVED** [review]

**Comment by ethantkoenig** [comment]
LGTM

**Pull request merged**